ote/src/cljs/ote/ui/form_fields.cljs

```diff
@@ -532,3 +532,17 @@
                   :style style}]
     (when error
         (tr [:common-texts :required-field]))])
+
+(defmethod field :brokerage-help-checkbox [{:keys [update! label warning error style help-text help-link-text help-link]} checked?]
```

Comment:
I would prefer that a normal `:checkbox` type be added with an `:extended-help` option or something instead of creating a new field type that is only used once.